openlibrary/macros/RecentPageList.html

```diff
@@ -5,19 +5,19 @@
 $ pages = list_recent_pages(path, limit=limit, offset=page * limit)
 
 <table>
-$for p in pages:
+    $for p in pages:
```

Comment:
These changes seem unrelated to the change being made. Indentation shouldn't have any impact on HTML validation.

I'd suggest creating a patch on master e.g. `git diff origin/master HTML_validation > patch.diff` and then carefully applying it on a fresh branch removing unrelated changes like these.

Reply:
Thank you for telling me this, I will do this, and for now, I will recheck all the things that I changed, I accidentally did some single commas to double commas, and I will try to resolve these